src/huggingface_hub/_commit_api.py

```diff
@@ -360,11 +359,12 @@ def _upload_lfs_files(
     for chunk in chunk_iterable(additions, chunk_size=256):
         batch_actions_chunk, batch_errors_chunk = post_lfs_batch_info(
             upload_infos=[op.upload_info for op in chunk],
-            token=token,
             repo_id=repo_id,
             repo_type=repo_type,
             revision=revision,
             endpoint=endpoint,
+            headers=headers,
+            token=None,  # already passed in 'headers'
```

Comment:
Should the `token` be generally expected in the headers in that case, rather than put as a kwarg? Or is it a public-facing function and we can't change the API maybe

Reply:
Yes I removed `token` whenever possible but this one is not a private method so just in case I did not remove `token` from the params.

Reply:
(though it's not a documented / promoted method)